bin/crontabScripts/autoUpdateRepo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#:Title: autoUpdateRepo.sh
+#:Author: Alan Coleman
+
+#changes directory to folder containing repositories
+cd gitfolder
+
+#runs repository update script 
+bash ../ELK_POC_HOME/elk-poc/GitUpdateScript.sh
```

Comment:
This seems to be an absolute path. Can you use env variable? $ELK_POC_HOME so that this can be run from anywhere.

Reply:
you might want to investigate using a system wide environment variable, as ELK_POC_HOME is only set for the current user (from the .bash_profile file)

Reply:
when i make the path's environmental variables, they run fine when the euser runs them, but not when they run via crontab